clients/extension/src/navigation/index.ts

```
@@ -0,0 +1,94 @@
+      }[keyof T & string]
+    : ''
+
+export type AppPath = DotNestedKeys<AppPaths>
+
+// TODO: to change as required in the extension
+export type AppPathParams = {
+  uploadQr: { title?: string }
+  import: { from?: string }
+}
+
+export type AppPathState = {}
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)